src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml

```diff
@@ -109,6 +110,11 @@
         </service>
         <service id="Sylius\Component\Core\Promotion\Applicator\UnitsPromotionAdjustmentsApplicatorInterface" alias="sylius.promotion.units_promotion_adjustments_applicator" />
 
+        <service id="sylius.promotion.additional_filters_applicator" class="Sylius\Component\Core\Promotion\Applicator\AdditionalFiltersApplicator">
+            <argument type="tagged_iterator" tag="sylius.promotion_additional_item_filters" />
+        </service>
+        <service id="Sylius\Component\Core\Promotion\Applicator\AdditionalFiltersApplicatorInterface" alias="sylius.promotion.additional_filters_applicator" />
```

Comment:
```suggestion
        <service id="Sylius\Component\Core\Promotion\Applicator\AdditionalFiltersApplicatorInterface" class="Sylius\Component\Core\Promotion\Applicator\AdditionalFiltersApplicator">
            <argument type="tagged_iterator" tag="sylius.promotion_additional_item_filters" />
        </service>
```

As this is a new service, FQCN is enough. This would follow Sylius' naming conventions https://docs.sylius.com/en/1.12/book/contributing/code/conventions.html?highlight=fqcn#naming

Reply:
ok I hadn't seen this page and I thought the best practice was to "name" the services